commands/system/help.js

```diff
@@ -1,83 +1,162 @@
 const Command = require(`${process.cwd()}/base/Command.js`);
 const { MessageEmbed } = require("discord.js");
+const EMOJIS = ["⏮", "◀", "⏹", "▶", "⏭", "🔢"];
```

Comment:
This variable should not be in full caps `emojis` will do

Reply:
@MrJacz kyra said to do `EMOJIS`

Reply:
Why? (╯°□°）╯︵ ┻━┻

Reply:
It's a convention for constants. I guess most things here are technically constants, though...